iocage_test.yml

```diff
@@ -1,90 +1,591 @@
 ---
-- hosts:
-    - localhost
-
+- hosts: test_iocage
```

Comment:
consider keeping `localhost` here, it'll be a lot easier for most of us to test it without a host/VM called `test_iocage`

Reply:
done